src/cli/api/local_api.py

```diff
@@ -36,24 +54,140 @@ def __is_config_valid(self, application: str) -> bool:
+
+        experiment_directory.mkdir(parents=True)
+
+        input_directory = experiment_directory / 'input'
+        input_directory.mkdir(parents=True)
+        self.__copy_input_files_from_application(application, input_directory)
+
+        experiment_json_file = experiment_directory / 'experiment.json'
+        experiment_meta = {
+            "backend": {
+                "location": "local",
+                "type": "local_netsquid"
+             },
+            "number_of_rounds": 1,
+            "description": f"{name}: experiment description"
+        }
+
+        asset_application = self.__create_asset_application(app_config)
+        asset_network = self.__create_asset_network(network, app_config)
+        asset = {"network": asset_network, "application": asset_application}
+
+        experiment_data = {'meta': experiment_meta, 'asset': asset}
+        write_json_file(experiment_json_file, experiment_data)
+
+        return True, "Success"
 
-    def run_experiment(self, path: str, block: bool) -> Optional[List[ResultType]]:
+    def __create_asset_application(self,  app_config: AppConfigType) -> List[Dict[str, Any]]:
+        return []
+
+    def __create_asset_network(self,  network: str, app_config: AppConfigType) ->  Dict[str, Any]:
+        return {}
+
+    def __copy_input_files_from_application(self,  application: str, input_directory: Path) -> None:
+        """
+        Copy the input/source files of the 'application' to the 'input_directory'
+
+        Args:
+            application: The application name for which the input files need to be copied
+            input_directory: The destination where application files need to be stored
+
+        """
+
+
+    def check_valid_network(self, network: str, app_config: AppConfigType) -> bool:
+        """
+        Check if the network name is a valid network for the Application
+
+        Args:
+            network: The network name
+            app_config: Application Configuration containing the available networks
+
+        Returns:
+            bool: True if the given network name is available in application configuration, False otherwise
+        """
+        return True
+
+    def is_experiment_local(self, path: Path) -> bool:
+        """
+        Check if the experiment at the 'path' is remote or local
+
+        Args:
+            path: The location of the experiment
+
+        Returns:
+            bool: True if the experiment at given path is local, False otherwise
+        """
+        # read experiment.json and check the meta->backend->location attribute
+        return True
+
+    def get_experiment_rounds(self, path: Path) -> int:
+        """
+        Get the number of rounds in an experiment
+
+        Args:
+            path: The location of the experiment
+
+        Returns:
+            int: Value of the number of rounds
+        """
+        # read experiment.json and check the meta->number_of_rounds attribute
```

Comment:
As discussed, what if we re-run the experiment. We will get a new round set.
So an experiment can have multiple times number of rounds. When they are all added (summed) to experiment we can return the total. Is that what we want here?